tests/testthat/reporters/check.txt

```diff
@@ -14,38 +14,38 @@ FALSE isn't true.
 
 ── 5. Error: Error:1 (@tests.R#33)  ────────────────────────────────────────────
 stop
-1: stop("stop") at reporters/tests.R:33
 
 ── 6. Error: Error:3 (@tests.R#47)  ────────────────────────────────────────────
 !
-1: f() at reporters/tests.R:47
-2: g() at reporters/tests.R:38
-3: h() at reporters/tests.R:41
-4: stop("!") at reporters/tests.R:44
+Backtrace:
+ 1. f()
+ 2. g()
+ 3. h()
 
 ── 7. Error: Recursion:1 (@tests.R#56)  ────────────────────────────────────────
 This is deep
-1: f(25) at reporters/tests.R:56
-2: f(x - 1) at reporters/tests.R:54
-3: f(x - 1) at reporters/tests.R:54
-4: f(x - 1) at reporters/tests.R:54
-5: f(x - 1) at reporters/tests.R:54
-6: f(x - 1) at reporters/tests.R:54
-7: f(x - 1) at reporters/tests.R:54
-8: f(x - 1) at reporters/tests.R:54
-9: f(x - 1) at reporters/tests.R:54
-10: f(x - 1) at reporters/tests.R:54
-...
-18: f(x - 1) at reporters/tests.R:54
-19: f(x - 1) at reporters/tests.R:54
-20: f(x - 1) at reporters/tests.R:54
-21: f(x - 1) at reporters/tests.R:54
-22: f(x - 1) at reporters/tests.R:54
-23: f(x - 1) at reporters/tests.R:54
-24: f(x - 1) at reporters/tests.R:54
-25: f(x - 1) at reporters/tests.R:54
-26: f(x - 1) at reporters/tests.R:54
-27: stop("This is deep") at reporters/tests.R:54
+Backtrace:
+  1. f(25)
```

Comment:
Why are we losing the location of the call? I realize these are noisy, but it is often useful information, and I typically do have the source references for all packages I compile on my machine (I have `R_KEEP_PKG_SOURCE=true` in my `.Renviron`).

Reply:
Only in the tests because foreign srcrefs now leak into the backtraces (e.g. from `rlang::eval_bare()`. The srcrefs are disabled with:

```r
withr::local_options(list(rlang_trace_format_srcrefs = FALSE))
```